bldr/cli.py

```diff
@@ -115,6 +115,12 @@ def _add_common_arguments(cls, parser: argparse.ArgumentParser) -> None:
             help="Disable using tmpfs.",
             action="store_true",
         )
+        parser.add_argument(
+            "--docker-timeout",
+            help="Default timeout for docker API calls, in seconds. (default: %(default)s)",
```

Comment:
Calling it "default" implies that it is or can be overridden somewhere, but I can't find any traces of that. Please drop the "default" qualifier.